tensorflow/examples/tutorials/word2vec/word2vec_basic.py

```diff
@@ -63,6 +63,7 @@ def read_data(filename):
 
 
 def build_dataset(words):
+  global vocabulary_size
```

Comment:
Avoid using global variables. Prefer making it into an argument of build_dataset.